platform/platform-app/pom.xml

```diff
@@ -317,6 +317,23 @@
             <artifactId>platform-email-impl</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>ddf.platform</groupId>
+            <artifactId>branding-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>ddf.platform</groupId>
+            <artifactId>ddf-branding-plugin</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.karaf.features</groupId>
+            <artifactId>standard</artifactId>
+            <version>${karaf.version}</version>
+            <type>xml</type>
+            <classifier>features</classifier>
```

Comment:
It seems odd that you'd need this `<classifier>features</classifier>` dependency here for `standard` but you don't need the same kind of dependency for `pax-cdi-features` and `spring`.